Symfony/CS/Fixer/Contrib/ShortEchoTagFixer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vincent Klaiber <hello@vinkla.com>
+ */
+class ShortEchoTagFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens as $index => $token) {
+            if ('<?=' !== $token->getContent()) {
```

Comment:
* Use `T_OPEN_TAG_WITH_ECHO` for first `if`
* Missing `T_WHITESPACE` between `T_OPEN_TAG` and `T_ECHO`
* Wrong index (`$index + 1`) for whitespace after `T_ECHO` (indexes have changed by the insertion
* use `for` instead of `foreach` and iterate backwards